### PR TITLE
feat: adding prop to datepicker for setting relative date range

### PR DESCRIPTION
**Pull request by jelleag**
#130

### CODE REVIEW

**Comment by severinlandolt** [comment]
Hey @jelleag, thank you for your constructive feedback. We will discuss your idea internally and get back to you soon.

File: src/components/input-elements/Datepicker/Datepicker.tsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ import Modal from 'components/layout-elements/Modal';
 export interface DatepickerProps {
     handleSelect?: { (selectedStartDay: Date, selectedEndDay: Date): void },
     enableRelativeDates?: boolean,
+    relativeFilterOption?: 'w' | 't' | 'm' | 'y' | null,
```
Comment (mitrotasios) [inline src/components/input-elements/Datepicker/Datepicker.tsx]: I think this suggestion makes sense, thanks @jelleag! Would just recommend to call the prop `defaultRelativeFilterOption` to maintain consistency.

File: src/components/input-elements/Datepicker/Datepicker.tsx [hunk 2]
```diff
@@ -78,7 +80,8 @@ const Datepicker = ({
     const [showDatePickerModal, setShowDatePickerModal] = useState(false);
     const [showDropdownModal, setShowDropdownModal] = useState(false);
 
-    const [selectedRelativeFilterOption, setSelectedRelativeFilterOption] = useState<string | null>(null);
+    const [selectedRelativeFilterOption, setSelectedRelativeFilterOption] = useState<string | null>(
```
Comment (mitrotasios) [inline src/components/input-elements/Datepicker/Datepicker.tsx:87]: Let's also account for the case where `enableRelativeDates=false`. In that case users shouldn't be able to set a `defaultRelativeFilterOption` :)

**Review by mitrotasios — CHANGES_REQUESTED** [review]

**Comment by jelleag** [comment]
Thanks for your comments! I've processed your suggestions in my latest commit

**Comment by mitrotasios** [comment]
Awesome, thanks! that was fast :D I will work on some other bug fixes the next 1-2 days and then merge your PR over the weekend hopefully, because this will trigger a minor release :)

**Review by mitrotasios — DISMISSED** [review]

**Review by mitrotasios — APPROVED** [review]

**Pull request merged**